pkg/watches/watches.go

```diff
@@ -98,6 +98,11 @@ func LoadReader(reader io.Reader) ([]Watch, error) {
 			w.WatchDependentResources = &trueVal
 		}
 
+		// make sure we do not have a nil LabelSelector. Instead just make it default to empty
+		if w.Selector == nil {
+			w.Selector = &metav1.LabelSelector{}
+		}
+
```

Comment:
Seems unrelated to this PR. Should we separate it into another PR?

Reply:
I was contemplating making a new issue and PR just for it but wasn't sure if I should. Will go ahead and do so.